_data/accessibility-tests/tooltip.yml

```diff
@@ -41,9 +55,7 @@ test_items:
 # Keyboard navigation tests
   - summary: Active tooltip content can be closed with a mouse or keyboard.
     summary_additional: When you open a tooltip with a mouse, you can use the "escape" key to close it.
```

Comment:
Format `escape` with backticks instead of quotes.

Reply:
Updated in [0c9c00e](https://github.com/uswds/uswds-site/pull/2646/commits/0c9c00e2f25f7fd9b408d8807ff55992aa5cbd07)